### PR TITLE
js forms

### PR DESCRIPTION
**Pull request by james-aguirre**
Screenshot 2023-04-05 at 10 58 20 AM

### CODE REVIEW

File: javascript-forms/form-events.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+$message.addEventListener('input', handleInput);
+
+function handleFocus(event) {
+  console.log('focus event fired');
+  console.log('event.target.name ', event.target.name);
+}
+
+function handleBlur(event) {
+  console.log('blur event fired target: ', event.target.name);
+}
```
Comment (shawnkost) [inline javascript-forms/form-events.js:24]: You're missing one console log here! You should have a console log saying 'blur event fired'

File: javascript-forms/form-events.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+function handleSubmit(event) {
+  event.preventDefault();
+  const messageData = {};
+  messageData.name = $form.name.value;
+  messageData.email = $form.email.value;
+  messageData.message = $form.message.value;
+  console.log(messageData);
+  $form.reset();
+}
```
Comment (shawnkost) [inline javascript-forms/form-events.js:41]: This should be in `form-values.js`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here James!

File: javascript-forms/form-events.html [hunk 3]
```diff
@@ -37,5 +37,6 @@ <h2>Contact Us</h2>
     </form>
   </div>
   <script src="form-events.js"></script>
+  <script src="form-values.js"></script>
```
Comment (shawnkost) [inline javascript-forms/form-events.html:40]: You shouldn't need to add this. You already have a `form-values.html` where this script tag is loaded

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change James!

File: javascript-forms/form-events.html (hunk 3 above)
Comment (james-aguirre) [reply javascript-forms/form-events.html]: Screenshot 2023-04-05 at 12 06 20 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**